po/de.po

```diff
@@ -335,7 +335,7 @@ msgstr "Initialisiert die Einstellungen weiter unten mit zufälligen Werten"
 
 #: settings/settings.ui:2607
 msgid "Generate Random Preset"
-msgstr "Zufällige Einstellungen"
+msgstr "Zufällige Voreinstellung erzeugen"
```

Comment:
I fear that this is a bit too long for that button. As the context is clear from the button's position, maybe
```suggestion
msgstr "Zufällig erzeugen"
```